install_and_test.sh

```diff
@@ -0,0 +1,5 @@
+eval $(opam env --switch=/home/user/goblint/analyzer --set-switch)
```

Comment:
The switch path is awfully specific. Should have at least a comment `# Replace with your ..` like below. But that still makes tracking the scripts with git painful, as everyone has their own path there.